Easydict/Feature/ViewController/View/Titlebar/EZTitlebar.m

```diff
@@ -64,7 +67,26 @@ - (void)setup {
         }
     }];
     
+    [self setupSettingButton];
+    
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(updateConstraints) name:EZQuickLinkButtonUpdateNotification object:nil];
+    
+}
+
+- (void)setupSettingButton {
+    EZOpenLinkButton *button = [[EZOpenLinkButton alloc] init];
+    NSImage *image = [[NSImage imageWithSystemSymbolName:@"switch.2" accessibilityDescription:nil]imageWithSymbolConfiguration:[NSImageSymbolConfiguration configurationWithScale:NSImageSymbolScaleLarge]];
+    
+    button.image = image;
+    self.settingButton = button;
+    button.clickBlock = nil;
```

Comment:
This settings button, should also allow users to hide it, just like the Google link button.